shell/components/BannerGraphic.vue

```diff
@@ -51,19 +51,6 @@ export default {
       data-testid="banner-title"
       class="title"
     />
-    <div
-      v-if="pref"
-      class="close-button"
-      data-testid="graphic-banner-close"
```

Comment:
It looks like `cypress/e2e/po/components/banner-graphic.po.ts` still contains a reference to this `data-testid`